python/dgl/contrib/data/knowledge_graph.py

```diff
@@ -410,8 +410,8 @@ def _load_data(dataset_str='aifb', dataset_path=None):
                 dst = nodes_dict[o]
                 assert src < num_node and dst < num_node
                 rel = relations_dict[p]
-                edge_list.append((src, dst, 2 * rel))
-                edge_list.append((dst, src, 2 * rel + 1))
+                edge_list.append((src, dst, 2 * rel + 1))
+                edge_list.append((dst, src, 2 * rel + 2))
```

Comment:
+1 +2?

Reply:
Yeah. Because relation id 0 is used for self relation... I will put a comment here. Interestingly the bug did not affect accuracy numbers...